platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/geometry/LatLng.java

```diff
@@ -118,6 +119,17 @@ public double getAltitude() {
         return altitude;
     }
 
+    /**
+     * Return a new LatLng object with a wrapped Longitude.  This allows original data object
+     * to remain unchanged.
+     * @return New LatLng object with wrapped Longitude
+     */
+    public LatLng wrap(){
+        LatLng wrappedVersion = new LatLng(this);
+        wrappedVersion.setLongitude(MathUtils.wrap(wrappedVersion.getLongitude(), GeoConstants.MIN_LONGITUDE, GeoConstants.MAX_LONGITUDE));
```

Comment:
Should we include a check similar to the one in telemetry (`if ((lon < GeoConstants.MIN_LONGITUDE) || (lon > GeoConstants.MAX_LONGITUDE)) {`) to avoid creating a new object if it's unnecessary?